x-pack/plugins/security_solution/public/network/components/kpi_network/common/index.tsx

```diff
@@ -59,9 +66,11 @@ export const NetworkKpiBaseComponent = React.memo<{
 
     return (
       <EuiFlexGroup wrap>
-        {statItemsProps.map((mappedStatItemProps) => (
-          <StatItemsComponent {...mappedStatItemProps} />
-        ))}
+        <CasesContext owner={[APP_ID]} userCanCrud={userCanCrud ?? false}>
+          {statItemsProps.map((mappedStatItemProps) => (
+            <StatItemsComponent {...mappedStatItemProps} showInspectButton={false} />
```

Comment:
`Warning: Each child in a list should have a unique "key" prop`... is this not showing here? do we need a key?